docs/content/preview/yugabyte-cloud/managed-freetrial.md

```diff
@@ -39,6 +34,11 @@ After the free trial begins, you can use your credits to create dedicated cluste
 - Partition by region
 - Read replicas
 
-Create a [billing profile](../cloud-admin/cloud-billing-profile/) and add a payment method any time. You won't be charged until your credits are used.
+[Choose a plan](https://www.yugabyte.com/yugabyte-cloud-standard-price-list/) any time. You won't be charged until your credits are used.
+
+## Limitations
 
-Only one free trial can be applied to an account.
+- Only one free trial can be applied to an account.
+- The free trial features the Professional plan. Enterprise plan features are not included.
+- Not all regions are available during free trials.
```

Comment:
@shreyas-agnihotri : do we also want to call out that we will only allow 24 cores maximum during the free trial period?